code/modules/mob/living/carbon/human/human.dm

```diff
@@ -416,15 +416,19 @@
 
 //Removed the horrible safety parameter. It was only being used by ninja code anyways.
 //Now checks siemens_coefficient of the affected area by default
-/mob/living/carbon/human/electrocute_act(var/shock_damage, var/obj/source, var/base_siemens_coeff = 1.0, var/def_zone = null, var/incapacitation_duration)
+/mob/living/carbon/human/electrocute_act(var/shock_damage, var/obj/source, var/base_siemens_coeff = 1.0, var/def_zone = null, var/incapacitation_duration, var/ignore_siemens = 0)
 	if(status_flags & GODMODE || (M_NO_SHOCK in src.mutations))
 		return 0	//godmode
 
 	if (!def_zone)
 		def_zone = pick(LIMB_LEFT_HAND, LIMB_RIGHT_HAND)
 
 	var/datum/organ/external/affected_organ = get_organ(check_zone(def_zone))
-	var/siemens_coeff = base_siemens_coeff * get_siemens_coefficient_organ(affected_organ)
+	var/siemens_coeff 
+	if(ignore_siemens)
```

Comment:
1. I don't think this realistically makes sense to pierce gloves if it's targeting hands
2. If your logic is "this will shock their exposed chest/head" why not target those instead? then it will use the coeff for those

Reply:
It does target the chest, the reason why I added the ignore parameter was because there were some clothing items like the magus robe/helm that had a siemens coeff of 0, which would make them completely immune to the giant ball of lightning

Reply:
That sounds like a feature to me

>lightning arcs to where you're protected
>it absorbs the shock

Reply:
fair enough, I'll change it back, after all there aren't many chest items with a siemens of 0, I just found it a little wierd that some things like cult robes gave you immunity to it

Reply:
deity....